open-api/src/testFixtures/java/org/apache/iceberg/rest/RESTCatalogServer.java

```diff
@@ -64,7 +72,8 @@ public Map<String, String> configuration() {
 
   private CatalogContext initializeBackendCatalog() throws IOException {
     // Translate environment variables to catalog properties
-    Map<String, String> catalogProperties = RCKUtils.environmentCatalogConfig();
+    Map<String, String> catalogProperties = Maps.newHashMap(config);
+    catalogProperties.putAll(RCKUtils.environmentCatalogConfig());
```

Comment:
I think we might want to turn the maps around, so that the initial map comes from `RCKUtils.environmentCatalogConfig()` and then we'd do `catalogProperties.putAll(config)`. That way we can override stuff via `config`

Reply:
sounds good. I saw you already handled this through your suggested commit. I adopted your code change.

Reply:
I actually handled something else. So this still needs to be done

Reply:
Done.